# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_util-linux.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some hackery, mostly even semi-valid fixes.

  - Rene Rebe <rene@exactcode.de>

--- util-linux-2.12q/mount/mount_by_label.c.vanilla	2005-06-16 01:20:52.000000000 +0200
+++ util-linux-2.12q/mount/mount_by_label.c	2005-06-16 01:21:59.000000000 +0200
@@ -189,7 +189,7 @@
 	if (!procpt) {
 		static int warn = 0;
 		if (!warn++)
-		    error (_("%s: could not open %s, so UUID and LABEL "
+		    fprintf (stderr, _("%s: could not open %s, so UUID and LABEL "
 			     "conversion cannot be done.\n"),
 			   progname, PROC_PARTITIONS);
 		return;
--- util-linux-2.12q/mount/fstab.c.vanilla	2005-06-16 01:19:50.000000000 +0200
+++ util-linux-2.12q/mount/fstab.c	2005-06-16 01:20:00.000000000 +0200
@@ -400,7 +400,7 @@
 static int signals_have_been_setup = 0;
 
 /* Ensure that the lock is released if we are interrupted.  */
-extern char *strsignal(int sig);	/* not always in <string.h> */
+//extern char *strsignal(int sig);	/* not always in <string.h> */
 
 static void
 handler (int sig) {
--- util-linux-2.12q/text-utils/column.c.vanilla	2005-06-16 01:23:44.000000000 +0200
+++ util-linux-2.12q/text-utils/column.c	2005-06-16 01:24:00.000000000 +0200
@@ -41,6 +41,8 @@
 #include <sys/types.h>
 #include <sys/ioctl.h>
 
+#include <termios.h>
+
 #include <ctype.h>
 #include <limits.h>
 #include <stdio.h>
--- util-linux-2.12q/partx/solaris.c.vanilla	2005-06-16 01:34:09.000000000 +0200
+++ util-linux-2.12q/partx/solaris.c	2005-06-16 01:34:15.000000000 +0200
@@ -5,8 +5,6 @@
 #define SOLARIS_X86_NUMSLICE	8
 #define SOLARIS_X86_VTOC_SANE	(0x600DDEEEUL)
 
-typedef int daddr_t;		/* or long - check */
-
 struct solaris_x86_slice {
 	unsigned short	s_tag;		/* ID tag of partition */
 	unsigned short	s_flag;		/* permision flags */
